webanno-ui-annotation/src/main/java/de/tudarmstadt/ukp/clarin/webanno/ui/annotation/dialog/AnnotationPreferenceModalPanel.html

```diff
@@ -65,7 +92,7 @@
 			</tr>
 			<tr class="border_bottom">
 				<td><label> Use the same color for all tags in a layer </label></td>
```

Comment:
I this label `Use the same color for all tags in a layer` still correct after the checkbox was changed to `readonlylayercoloring`?